usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -8731,6 +8731,7 @@ static const CK_MECHANISM_TYPE ep11_supported_mech_list[] = {
     CKM_SHA_1_HMAC,
     CKM_GENERIC_SECRET_KEY_GEN,
     CKM_IBM_ATTRIBUTEBOUND_WRAP,
+    CKM_GENERIC_SECRET_KEY_GEN,
```

Comment:
It was there, two lines above

Reply:
Arg, you are right. It was already added by https://github.com/opencryptoki/opencryptoki/commit/945a46aecd3006387e551a76d8d94ea622bddc4a
I guess this was a git-rebase error to add it again. 
SO I'll drop this commit entirely.